src/blocks/store/adl.clj

```diff
@@ -169,10 +191,26 @@
     (let [path (id->path root (:id block))]
       (when-not (.checkExists client path)
         (with-open [;output (.createFile client path IfExists/OVERWRITE "440" false)
-                    output (.createFile client path IfExists/FAIL)
+                    output (.createFile client path IfExists/FAIL write-permission false)
                     content (block/open block)]
-          (io/copy content output)))
-      ; TODO: set read-only bits to indicate we're done writing
+          (comment
+            "can hit this exception from time to time (~ 1/100 puts)
```

Comment:
If this whole thing is just going to be a string, you might as well make these semicolon comments. `comment` is more useful for eliding whole code forms that are useful in a dev or testing context.